metadata-models/src/main/pegasus/com/linkedin/glossary/GlossaryRelatedTerms.pdl

```diff
@@ -0,0 +1,54 @@
+namespace com.linkedin.glossary
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.GlossaryTermUrn
+
+/**
+ * Has A / Is A lineage information about a glossary Term reporting the lineage
+ */
+ @Aspect = {
+  "name": "glossaryRelatedTerms"
+}
+record GlossaryRelatedTerms {
+
+  /**
+   * Audit stamp containing who reported the lineage and when
+   */
+  auditStamp: AuditStamp
```

Comment:
what will be the source of this audit stamp? we want to be conscious about including these only if we can populate them. will you have source data for created and updated at?

Reply:
Removed the auditStamp. We can have this when we want to store the version history of glossaryTerm relationships.